File: laba3/lab3.php
<?php
function searchEven($num) {
    $count = 0;
    while ($num > 0) {
        $last = $num % 10;
        if ($last % 2 == 0) $count++;
        $num = intdiv($num, 10);
    }
    return $count;
}

echo "Enter number of numbers: ";
$n = intval(trim(fgets(STDIN)));

for ($i = 0; $i < $n; $i++) {
    echo "Enter number: ";
    $k = intval(trim(fgets(STDIN)));
    echo searchEven($k) . "\n";
}
?>

File: laba3/lab3.kt
fun searchEven(num: Int): Int {
    var n = num
    var count = 0
    while (n > 0) {
        val last = n % 10
        if (last % 2 == 0) count++
        n /= 10
    }
    return count
}
//https://www.tutorialspoint.com/compilers/online-kotlin-compiler.htm
fun main() {
    print("Enter number of numbers: ")
    val n = readLine()!!.toInt()
    repeat(n) {
        print("Enter number: ")
        val k = readLine()!!.toInt()
        println(searchEven(k))
    }
}
